browser/src/neovim/NeovimInstance.ts

```diff
@@ -651,6 +722,7 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
                 rgb: true,
                 popupmenu_external: true,
                 ext_tabline: shouldExtTabs,
+                ext_cmdline: true,
```

Comment:
We might want a configuration for this - something like `editor.commandLine.mode` with `'oni'` or `'native'` as options (kind of like what we have for `tabs.mode`, and `editor.comlpetions.mode`).